controls/core.sp

```diff
@@ -0,0 +1,397 @@
+locals {
```

Comment:
@Subhajit97 Can you please try renaming this benchmark and its controls from core->compute and see how that looks in the hub site? Core is the TF + table name scheme we use, but Compute seems to be more common in OCI's docs, so I'm interested to see if this one is more recognizable for users.